Example/AblyChatExample/ContentView.swift

```diff
@@ -3,22 +3,130 @@ import SwiftUI
 
 struct ContentView: View {
     /// Just used to check that we can successfully import and use the AblyChat library. TODO remove this once we start building the library
-    @State private var ablyChatClient = DefaultChatClient(
-        realtime: MockRealtime(key: ""),
-        clientOptions: ClientOptions()
-    )
-
+//    @State private var ablyChatClient = DefaultChatClient(
+//        realtime: MockRealtime(key: ""),
+//        clientOptions: ClientOptions()
+//    )
+    
+    var room: Room {
+        try! MockChatClient.shared.rooms.get(roomID: "Demo Room", options: .init())
+    }
+    
+    // compiler doesn't allow to use `room` property here, needs to think about proper initialization
+    @StateObject private var messagesClient = try! MockChatClient.shared.rooms.get(roomID: "Demo Room", options: .init()).messages as! MockMessages
```

Comment:
Why the `as! MockMessages`? This means that we’re no longer just using the API of the SDK (i.e. the `Messages` protocol).

Reply:
This is also forced by a compiler:
```
Type 'any Messages' cannot conform to 'ObservableObject'
```
But this is an experimental PR in which I'm just trying things. An expanded answer is in the https://github.com/ably-labs/ably-chat-swift/pull/9